main.go

```diff
@@ -304,7 +304,7 @@ func main() {
 			for key, value := range etcdSecret.Data {
 				if key == modelmesh.EtcdSecretKey {
 					var data map[string]json.RawMessage
-					err := json.Unmarshal(value, &data)
+					err = json.Unmarshal(value, &data)
```

Comment:
Easter egg? :)

Reply:
@danielezonca lol upstream lint check failed and I had to address it. But then we removed upstream lint check xD but I still kept the change.